src/groupeddatatable/grouping.jl

```diff
@@ -240,20 +240,12 @@ colwise(sum, groupby(dt, :a))
 """
 function colwise(f, d::AbstractDataTable)
     x = [f(d[i]) for i in 1:ncol(d)]
-    if eltype(x) <: Nullable
-        return NullableArray(x)
-    else
-        return x
-    end
+    return x
```

Comment:
No need for `x`, just make this a one-liner and use the short function definition syntax.Same below.